src/debug/ee/debugger.cpp

```diff
@@ -67,6 +67,8 @@ GVAL_IMPL_INIT(ULONG, CLRJitAttachState, 0);
 
 bool g_EnableSIS = false;
 
+bool g_ShouldTrackDynamicMethodDebugInfo = false;
```

Comment:
We should use EEConfig for caching settings like this. Here is an example showing [caching](https://github.com/dotnet/coreclr/blob/cb22707acb620e93a30254ded3048912a9ccb26f/src/vm/eeconfig.cpp#L741), [accessor](https://github.com/dotnet/coreclr/blob/b4390c41f29af20e4cb77d296940bb6c4649ee86/src/vm/eeconfig.h#L588), and [use](https://github.com/dotnet/coreclr/blob/58d9cf157f54e8fd61eaaf56b3f8045075d171cd/src/vm/finalizerthread.cpp#L468)